index.js

```diff
@@ -154,17 +154,42 @@ export default class ClearvisioAppointmentBooker {
   }
 
   async loadStore(storeCode) {
-    var stores = await api.get(this.store, `stores?code=${storeCode}`);
-    this.store.dispatch('store/set', stores[0]);
+    try {
+      var stores = await api.get(this.store, `stores?code=${storeCode}`);
+      if ( stores[0] == undefined) {
+        this.store.dispatch('moduleState/set', 'error.storeCode');
+        return;
+      }
+      this.store.dispatch('store/set', stores[0]);
+    } catch (error) {
+      if (error.code == 403) {
+        this.store.dispatch('moduleState/set', 'error.403');
+      } else {
+        this.store.dispatch('moduleState/set', 'error.storeCode');
+      }
+    }
   }
 
   async loadEyeExaminationProcesses({eyeExaminationProcessId}) {
     if (eyeExaminationProcessId) {
-      return [await api.get(this.store, `eye_examination_processes/${eyeExaminationProcessId}`)]
-        .filter((process) => process);
+      try {
+        const  process = await api.get(this.store, `eye_examination_processes/${eyeExaminationProcessId}`);
+        return process
+      } catch(error) {
+        if (error.code == 404){
```

Comment:
egyrészt
```suggestion
        if (error.code == 404) {
```


másrészt: mi van akkor ha nem 404. ugye akkor valami nem várt hibánk van. ilyenkor illik tovább dobni / return-özni, vagy valami